content/schemas/sign_request_prefill_tag.yml

```diff
@@ -0,0 +1,28 @@
+---
+title: Sign Request Prefill Tag
+
+type: object
+
+description: |-
+  Prefill tags used to prefill signer input data
+
+properties:
+  external_id:
```

Comment:
Can it be called "tag_id" since it's for tags?  external_id implies it's a free-form field